ampd/src/config.rs

```diff
@@ -591,4 +568,22 @@ mod tests {
             ..Config::default()
         }
     }
+
+    #[test]
+    fn deserialize_monitoring_server_config() {
+        let bind_address = "0.0.0.0:3001";
+        let config_str = format!(
+            "
+            [monitoring_server]
+            enabled = true
+            bind_address = '{bind_address}'
+            ",
+        );
+        let cfg: Config = toml::from_str(&config_str).unwrap();
+        assert!(cfg.monitoring_server.enabled);
+        assert_eq!(
+            cfg.monitoring_server.bind_address,
+            SocketAddrV4::new(Ipv4Addr::new(0, 0, 0, 0), 3001)
+        );
+    }
```

Comment:
can you also add a test where the bind_address is not defined, so we can make sure it won't break deserialization?